data/human/intro missions.txt

```diff
@@ -408,7 +408,7 @@ mission "Intro [2 Interceptor]"
 	on complete
 		payment 40000
 		conversation
-			`As you land on Ingot, James says, "Well, you made it in one piece! Still think you want to be piloting a Sparrow?"`
+			`As you land on Ingot, James says, "Well, you made it in one piece! Still think you want to be piloting an interceptor?"`
```

Comment:
It feels like "an interceptor" would be better than "a Sparrow" because there are some pretty good interceptors.
This is only a minor thing though and the tweaks seem mostly good.

Reply:
I can make it say sparrow when its a sparrow if you want, and otherwise interceptor

Reply:
Actually interceptor is fine. Honestly, you only needed to take a short break to obtain a Hawk so you might have a powerful interceptor.